platform/solr/solr-factory/src/main/java/org/codice/solr/factory/impl/HttpSolrClientFactory.java

```diff
@@ -109,7 +109,7 @@
    * @return Solr server secure HTTP address
    */
   public static String getDefaultHttpsAddress() {
-    return SystemBaseUrl.constructUrl("https", SOLR_CONTEXT);
+    return SystemBaseUrl.INTERNAL.constructUrl("https", SOLR_CONTEXT);
```

Comment:
❓@paouelle, @pklinef, should we be using the external URL here instead? Does this still even matter?

Reply:
The default should continue to go to the internal one. Otherwise, the customer has to configure it otherwise. That being said, this code is completely modified by the following PR (https://github.com/codice/ddf/pull/3031) which is scheduled to be merged into master this sprint.